index.html

```diff
@@ -5026,6 +5026,24 @@ <h2>Level of Assurance</h2>
 href="#privacy-considerations"></a> is applicable for such extensions.
       </p>
     </section>
+
+    <section class="informative">
+      <h2>Evaluating Competing Considerations</h2>
+
+      <p>
+This specification does not require or suggest the use of any specific type of
+<a>verifiable data registry</a>. Different use cases might result in different
+requirements. Different requirements might suggest different considerations with different
+trade-offs. For example, trade-offs between computation (energy usage), trust
+(deference to authority), coordination (network bandwidth), or memory (physical
+storage) might or might not be appropriate, for any given use case. Other use cases
+might not make the same trade-offs. Those that need to consider different criteria
```

Comment:
```suggestion
might not make the same trade-offs. Those that need to consider different factors
```

Reply:
Skipping this suggestion since the DID Rubric calls these things "criteria": https://www.w3.org/TR/did-rubric/#categories-of-criteria

Reply:
I'll live with it. Constructions like "those that need x should look at y which provides x" just make me itch. You can mark this as resolved if Github lets you. (It doesn't give me that option.)